utils/account-setup/index.js

```diff
@@ -0,0 +1,139 @@
+    })
+}
+
+
+async function main () {
+
+    // BEGIN - Connect to XRPL API 
+    const xrplApi = new XrplApi(RIPPLED_URL);
+    await xrplApi.connect();
+    // END - Connect to XRPL API
+
+    try {       
+
+        // BEGIN - Account Creation 
+        console.log('Started to create XRP Accounts');
+        const unresolved = accounts.map(async function(item) {
+            const resp = await httpPost(FAUCETS_URL);           
+            const json = JSON.parse(resp);
+            const acc = new XrplAccount(json.account.address, json.account.secret, { xrplApi: xrplApi });
+
+            if (item === "ISSUER") {
+                await new Promise(r => setTimeout(r, 2000));
+                await acc.setDefaultRippling(true);
+                console.log('Enabled Rippling in ISSUER Account');
+            }
+
+            console.log(`Created ${item} Account`);
+
+            return { "name" : item, "xrplAcc": acc };
+            
+          });
+
+        const newAccounts = await Promise.all(unresolved);
+        // END - Account Creation
+
+        // 5 second wait in order to sync the accounts
+        await new Promise(r => setTimeout(r, 5000));
+        
+        // BEGIN - Trust Lines initiation
+        const foundation_lines = await newAccounts[1].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (foundation_lines.length === 0) {
+            await newAccounts[1].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, TOTAL_MINTED_EVRS);
+        }
+
+        const comm_bank_lines = await newAccounts[2].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (comm_bank_lines.length === 0) {
+            await newAccounts[2].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, PURCHASER_PROGRAM_EVRS);
+        }
+
+        const registry_lines = await newAccounts[3].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (registry_lines.length === 0) {
+            await newAccounts[3].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, TOTAL_MINTED_EVRS);
+        }
+
+        const purchaser_lines = await newAccounts[4].xrplAcc.getTrustLines(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        if (purchaser_lines.length === 0) {
+            await newAccounts[4].xrplAcc.setTrustLine(evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address, PURCHASER_PROGRAM_EVRS);
+        }
+
+        console.log("Trust Lines initiated");
+        // END - Trust Lines initiation
+
+        // BEGIN - Transfer Currency
+        await newAccounts[0].xrplAcc.makePayment(newAccounts[1].xrplAcc.address, TOTAL_MINTED_EVRS, evernode.EvernodeConstants.EVR, newAccounts[0].xrplAcc.address);
+
+        console.log(`${TOTAL_MINTED_EVRS}/- EVR amount was issued to EVERNODE Foundation`);
```

Comment:
Let's make this `${TOTAL_MINTED_EVRS} EVRs was issued to EVERNODE Foundation account` for more readability
Do this in the below logs as well